### PR TITLE
Making url

### PR DESCRIPTION
**Pull request by arno-hartmann**
Made lambda write Destination and booking link to flask.

### CODE REVIEW

File: get_flight/src/get_flight_url_utils.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    response = table_destination.scan()
+    data = response['Items']
+    while 'LastEvaluatedKey' in response:
+        response = table_destination.scan(ExclusiveStartKey=response['LastEvaluatedKey'])
+        data.extend(response['Items']) 
+    return data 
+
+
+def write_write_url_to_dynamo(city, city_id, url):
+    table_destination = dynamodb.Table('destination')
```
Comment (fabianschmauder) [inline get_flight/src/get_flight_url_utils.py]: you define the multiple times in this file.
maybe move name or table to constant or

File: get_flight/variables.tf [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+  
+variable "sunny_table_name" {
+
```
Comment (fabianschmauder) [inline get_flight/variables.tf]: you can add the type here

File: setup_sunnydays.sh [hunk 3]
```diff
@@ -22,8 +26,9 @@ terraform init
 terraform plan
 terraform apply
 
-sleep 20
 echo "waiting for 20 seconds"
```
Comment (fabianschmauder) [inline setup_sunnydays.sh]: why? 
maybe wait 20 seconds for instances to get ready

File: get_flight/src/get_flight_url_utils.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+def startdate():
+    return date.today() + timedelta(days =1)
+
+def returndate(start_date, trip_length):
+    return start_date + timedelta(days = trip_length)
+
+def maketimestring(day):
+    return day.strftime("%y%m%d")
+    return date
```
Comment (fabianschmauder) [inline get_flight/src/get_flight_url_utils.py]:
```suggestion
```

**Review by fabianschmauder — COMMENTED** [review]
small changes :)

File: setup_sunnydays.sh (hunk 3 above)
Comment (arno-hartmann) [reply setup_sunnydays.sh]: Done!

File: get_flight/variables.tf (hunk 2 above)
Comment (arno-hartmann) [reply get_flight/variables.tf]: Done

**Pull request merged**